tests/test-docker-image.sh

```diff
@@ -2,7 +2,7 @@
 
 # Test Docker image
 #
-# Usage: test-docker-image.sh TAG
+# Usage: test-docker-image.sh REPO/IMAGE:TAG
```

Comment:
Just made this modification in case folks in the future want to work on the CICD besides me. Will prevent this file from having to be edited when you have contributors.